storage/mdadm/files/udev-md-raid-arrays.rules

```diff
@@ -0,0 +1,36 @@
+# do not edit this file, it will be overwritten on update
+
+SUBSYSTEM!="block", GOTO="md_end"
```

Comment:
does these comes from upstream, then we can avoid putting it here and se the upstream ones as is

Reply:
We cannot use udev-rules as is... Most of them expect shell and systemd in the system.
I've remove unnecessary rules. We did the same in eudev module too...

Reply:
okay, makes sense